runtime/src/iree/hal/drivers/local_task/task_command_buffer.c

```diff
@@ -541,7 +541,7 @@ static iree_status_t iree_hal_task_command_buffer_fill_buffer(
       /*z=*/1,
   };
   const uint32_t workgroup_count[3] = {
-      /*x=*/length / workgroup_size[0] + 1,
+      /*x=*/iree_device_size_ceil_div(length, workgroup_size[0]),
```

Comment:
Curious: the old code wasn't doing a `ceil_div(x,y)`, it was doing a `x/y + 1`. Was it incorrect in that way too? Or just approximative in a way that was OK?

Reply:
It was doing one more workgroup than it needed in cases where the bytes-per-workgroup evenly divided the copy length, which would early-out inside itself. This now only does the extra workgroup if there's a need so it saves a bit of overhead. Silly laziness on my part when I originally wrote it :)